src/algorithm/extremes.rs

```diff
@@ -11,11 +10,6 @@ use types::{Extremes, ExtremePoint};
 // -1, 0. = smallest x
 
 // various tests for vector orientation relative to a direction vector u
-fn up<T>(u: &Point<T>, v: &Point<T>) -> bool
```

Comment:
Yep, but something like "is vector v pointing up, relative to vector u" (as per the [original](http://geomalgorithms.com/a14-_extreme_pts.html#polyMax_2D())) is fine, just in case.